en-US/snippets/2017/oct2017.lang

```diff
@@ -0,0 +1,99 @@
+
+
+# Snippet for Firefox
+;Firefox is the only browser that’s made by a non-profit, so it doesn’t sell access to your every click. #Nowyouknow
+Firefox is the only browser that’s made by a non-profit, so it doesn’t sell access to your every click. #Nowyouknow
+
+
+# Snippet for Firefox
+;Mozilla is a champion of the open web. Firefox is made by Mozilla. So every time you use Firefox, (like right now) you’re helping to make the web better for everyone.
+Mozilla is a champion of the open web. Firefox is made by Mozilla. So every time you use Firefox, (like right now) you’re helping to make the web better for everyone.
+
+
+# Snippet for form fill feature
+# Link: https://support.mozilla.org/kb/control-whether-firefox-automatically-fills-forms
+;Firefox can <a>fill out forms for you,</a> so you have more time to do important stuff online, like look at memes.
+Firefox can <a>fill out forms for you,</a> so you have more time to do important stuff online, like look at memes.
+
+
+# Snippet for add-ons
+# Link: https://addons.mozilla.org/firefox/collections/mozilla/essential-extensions/
+;Powerful privacy tools, media enhancers and more, our Essential extensions collection is loaded with awesome add-ons. <a>Check them out.</a>
+Powerful privacy tools, media enhancers and more, our Essential extensions collection is loaded with awesome add-ons. <a>Check them out.</a>
+
+
+# Snippet for add-ons
+# Fidget spinners are the lastest craze in the US. Feel free to swap out with something that's very popular locally at the moment.
+# Link: https://addons.mozilla.org/firefox/collections/mozilla/trending-add-ons/
+;Unlike your fidget spinner, you’ll probably still have a use for one of our trending add-ons in a month. <a>Give one a try.</a>
+Unlike your fidget spinner, you’ll probably still have a use for one of our trending add-ons in a month. <a>Give one a try.</a>
+
+
+# Snippet for add-ons
+# Here add-on collection is humorously tied with trendy wardrobes.
+# Link: https://addons.mozilla.org/firefox/collections/mozilla/trending-add-ons/
+;Don't let your friends see you with an outdated add-on. <a>Explore our trending add-ons collection</a> for the latest in excellent Firefox extensions.
+Don't let your friends see you with an outdated add-on. <a>Explore our trending add-ons collection</a> for the latest in excellent Firefox extensions.
+
+
+# Snippet for add-ons
+# "Take one for a spin" means to try something out.
+# Link: https://addons.mozilla.org/firefox/collections/mozilla/trending-add-ons/
+;Our trending extensions collection is filled with emerging add-ons that are gaining traction among Firefox users. <a> Take one for a spin.</a>
+Our trending extensions collection is filled with emerging add-ons that are gaining traction among Firefox users. <a> Take one for a spin.</a>
+
+
+# Snippet on Send Tabs
+# Roam here means mobility between devices.
```

Comment:
Not sure this comment is helpful, it suggests to lose the "tie down vs roam free" part.